images/parity_parity/src/image.rs

```diff
@@ -0,0 +1,65 @@
+pub struct ParityEthereum {
+    arguments: ParityArgs,
+}
+
+#[derive(Clone)]
+pub struct ParityArgs {
+    pub unsafe_expose: bool,
+    pub jsonrpc_apis: String,
+    pub config: String,
```

Comment:
Could we make this an enum instead?

Reply:
We don't want to make it an enum because it's too brittle. If they add a new config type a user of this will be able to use that new config value. It's better to just leave it as a stirng. 

On the other hand we made jsonrpc_apis a Vec<String> since it is a list of strings separated by `,`.

Reply:
Well, the purpose of the testcontainers library is to facilitate testing. It is not intended to be used to run a node against mainnet. Thus I would make an enum here with all the Config values that currently make sense. If that is only `dev` for now, it should only be possible to put that value there.